src/linear_algebra/general_linear_group.lean

```diff
@@ -166,4 +184,24 @@ lemma to_GL_pos_injective :
  from subtype.coe_injective).of_comp
 
 end special_linear_group
+
+section examples
+
+/-- The matrix [a, b; -b, a] (inspired by multiplication by a complex number); it is an element of
+`GL_2` if `a ^ 2 + b ^ 2` is nonzero. -/
+def plane_conformal_matrix {R} [field R] (a b : R) (hab : a ^ 2 + b ^ 2 ≠ 0) :
```

Comment:
```suggestion
@[simps {fully_applied := ff}]
def plane_conformal_matrix {R} [field R] (a b : R) (hab : a ^ 2 + b ^ 2 ≠ 0) :
```
Will generate the lemma below, and the other one we forgot

Reply:
@eric-wieser Regarding the so-called "forgotten" lemma, produced by `simps` as
```lean
matrix.coe_inv_plane_conformal_matrix {R} [field R] (a b : R) (hab : a ^ 2 + b ^ 2 ≠ 0) :
  ↑(plane_conformal_matrix a b hab)⁻¹ = ![![a, b], ![-b, a]]⁻¹
```
(1) The linter objects to this; (2) in any case I think the correct version would be
```lean
  ↑(plane_conformal_matrix a b hab)⁻¹ = (a^2 + b^2)⁻¹ • ![![a, -b], ![b, a]]
```
which has some content and therefore is beyond the scope of this PR.

Can you please suggest syntax to make `simps` omit this auto-generated lemma?

Reply:
I switched it back to the original version.

Reply:
`@[simps coe {fully_applied := ff}]` should generate only the wanted lemma

Reply:
OK, done